go/chat/inboxsource.go

```diff
@@ -1107,19 +1107,24 @@ func (s *localizerPipeline) localizeConversation(ctx context.Context, uid gregor
 	// channel information for a team chat
 	switch conversationRemote.GetMembersType() {
 	case chat1.ConversationMembersType_TEAM, chat1.ConversationMembersType_IMPTEAM:
+		var kuids []keybase1.UID
 		for _, uid := range conversationRemote.Metadata.AllList {
-			uname, err := uloader.LookupUsername(ctx, keybase1.UID(uid.String()))
-			if err != nil {
-				// If we are offline, we just won't know who is in the channel
-				continue
+			kuids = append(kuids, keybase1.UID(uid.String()))
+		}
+		unames, err := s.G().UIDMapper.MapUIDsToUsernames(ctx, s.G(), kuids)
+		if err == nil {
+			// If we are offline, we just won't know who is in the channel
```

Comment:
Is this comment still correct?  I think you'll get an `error` back if you're offline.

Reply:
I'll move it.